apps/web/public/js/formbricks-core.d.ts

```
@@ -0,0 +1,199 @@
+    data: ZResponseData,
+    variables: ZResponseVariables.optional(),
+    ttc: ZResponseTtc.optional(),
+    language: z.string().optional(),
+});
+
+declare const ZUploadFileConfig = z.object({
+    allowedFileExtensions: z.array(z.string()).optional(),
+    surveyId: z.string().optional(),
+});
+
+export { }
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)